File: an2_sem4/MPP/Homework/sapt11/java/network/src/main/java/ro/ubbcluj/cs/stefnmubb/festivalsellpoint/network/objectprotocol/LoginAngajatResponse.java
package ro.ubbcluj.cs.stefnmubb.festivalsellpoint.network.objectprotocol;

import ro.ubbcluj.cs.stefnmubb.festivalsellpoint.network.dto.AngajatDTO;

public class LoginAngajatResponse implements IResponse {
    private AngajatDTO angajat;

    public AngajatDTO getAngajat() {
        return angajat;
    }

    public LoginAngajatResponse(){}

    public LoginAngajatResponse(AngajatDTO angajat) {
        this.angajat = angajat;
    }


    @Override
    public String toString() {
        return "LoginAngajatResponse{" +
                "angajat=" + angajat +
                '}';
    }
}

File: an2_sem4/MPP/Homework/sapt10/cs/FestivalSellpoint/FestivalSellpoint.Network/ObjectProtocol/LoginAngajatResponse.cs
using FestivalSellpoint.Network.DTO;
using System;

namespace FestivalSellpoint.Network.ObjectProtocol
{
    [Serializable]
    public class LoginAngajatResponse : IResponse
    {
        public AngajatDTO Angajat { get; }
        public LoginAngajatResponse(AngajatDTO angajat) => Angajat = angajat;
    }
}
